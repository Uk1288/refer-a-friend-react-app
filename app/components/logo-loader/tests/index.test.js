import React from 'react';
import { render } from 'react-testing-library';
import { IntlProvider } from 'react-intl';
import LogoLoader from '../index';

describe('<LogoLoader />', () => {
  it('should render and match the snapshot', () => {
    const {
      container: { firstChild },
    } = render(
      <IntlProvider locale="en">
        <LogoLoader />
      </IntlProvider>,
    );
    expect(firstChild).toMatchSnapshot();
  });
});
