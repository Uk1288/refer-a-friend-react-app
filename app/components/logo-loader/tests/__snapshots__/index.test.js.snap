// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<LogoLoader /> should render and match the snapshot 1`] = `
<div>
  <div
    class="logo-loader__StyledImageParent-sc-1r2a1aj-1 brbqgv"
  >
    <img
      alt="Refer a friend logo"
      class="logo-loader__StyledImage-sc-1r2a1aj-0 dZGvMc"
      src="IMAGE_MOCK"
    />
  </div>
</div>
`;
